docs/bugpattern/ChainedAssertionLosesContext.md

```diff
@@ -16,7 +16,7 @@ After:
 ```
 class MyProtoSubject {
   public void hasFoo(Foo expected) {
-    check("foo()").that(actual().foo()).isEqualTo(expected);
+    check("foo()").that(actual.foo()).isEqualTo(expected);
```

Comment:
This example lacks an `actual` variable or field, so I wonder if it should be updated?

Reply:
Ditto.